src/style.css

```diff
@@ -0,0 +1,59 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+  outline: none;
+  font-family: "Avenir", sans-serif;
+  color: rgba(61, 78, 97, 1);
```

Comment:
move outline and color to special elements you need to change
don't apply it for all elements (`*`)